packages/react-styles/src/css/base/patternfly-svg.css

```diff
@@ -0,0 +1,5 @@
+@charset "UTF-8";
+
+.pf-svg-vertical-align {
```

Comment:
I don't know exactly where to put this CSS class. It would require manually importing this file in the every application for this to work properly.

Reply:
Yes, this belongs in https://github.com/patternfly/patternfly or existing applications will not get the styles.

Reply:
@wojta we have a global PF class stylesheet here where you could add this class https://github.com/patternfly/patternfly/blob/master/src/patternfly/base/_common.scss. Let me know if you'd like me to create the issue/PR.

Reply:
@mcooker added there https://github.com/patternfly/patternfly/pull/3775